src/grass_gis_helpers/open_geodata_germany/federal_state.py

```diff
@@ -86,7 +86,7 @@ def import_administrative_boundaries(output, aoi=None, level="KRS"):
         grass.run_command("g.region", vector=aoi, quiet=True)
 
     # url of administrative boundaries
-    URL = (
+    url = (
```

Comment:
Will `black` das nicht mehr als Konstante groß geschrieben haben?

Reply:
Es ist `pylint`, das meckert:
` src/grass_gis_helpers/open_geodata_germany/federal_state.py:89:4: C0103: Variable name "URL" doesn't conform to snake_case naming style (invalid-name)`